redturtle/bandi/upgrades.py

```diff
@@ -185,4 +185,14 @@ def migrate_to_2102(context):
             )
         )
         bando = brain.getObject()
-        bando.reindexObject(idxs=['tipologia_bando_label'])
+        bando.reindexObject(idxs=["tipologia_bando_label"])
+
+
+def migrate_to_2300(context):
+    PROFILE_ID = "profile-redturtle.bandi:to_2300"
+    context.runAllImportStepsFromProfile(PROFILE_ID)
+
+    #  update indexes and topics
```

Comment:
Is it needed this? You are importing to_2300 profile's registry.xml

Reply:
Yes it is